pkg/api/controller_test.go

```diff
@@ -110,12 +110,8 @@ func TestKVEnabled(t *testing.T) {
 	testControllerWithKV(t, true)
 }
 
-func TestKVDisabled(t *testing.T) {
-	testControllerWithKV(t, false)
-}
-
 func testController_ListRepositoriesHandler(t *testing.T, kvEnabled bool) {
-	clt, deps := setupClientWithAdmin(t, kvEnabled)
+	clt, deps := setupClientWithAdmin(t)
```

Comment:
I would love to see the module gets reverted back, so we will be able to run each test separately 😀

Reply:
https://github.com/treeverse/lakeFS/issues/4075